server/sonar-ce/src/main/java/org/sonar/ce/app/CeServer.java

```diff
@@ -138,6 +139,7 @@ public CeMainThread() {
     @Override
     public void run() {
       boolean startupSuccessful = attemptStartup();
+      this.operational = startupSuccessful;
```

Comment:
Can't you directly assign `this.operational = attemptStartup();` and remove startupSuccessful local variable?

Reply:
I could, but having the variable `startupSuccessful` strongly improved readability of this piece of code, doesn't it?

Since, eg., you did have trouble of this code, I believe anything helping understanding it is welcome :)

Reply:
As you want. On my side it was confusing, since I was looking for the use of this local variable.